RecoTracker/LSTCore/interface/ObjectRangesSoA.h

```diff
@@ -0,0 +1,51 @@
+#ifndef RecoTracker_LSTCore_interface_ObjectRangesSoA_h
+#define RecoTracker_LSTCore_interface_ObjectRangesSoA_h
+
+#include "DataFormats/SoATemplate/interface/SoALayout.h"
+#include "DataFormats/Portable/interface/PortableCollection.h"
+
+namespace lst {
+
+  GENERATE_SOA_LAYOUT(ObjectRangesSoALayout,
+                      SOA_COLUMN(ArrayIx2, hitRanges),
+                      SOA_COLUMN(int, hitRangesLower),
+                      SOA_COLUMN(int, hitRangesUpper),
+                      SOA_COLUMN(int8_t, hitRangesnLower),
+                      SOA_COLUMN(int8_t, hitRangesnUpper),
+                      SOA_COLUMN(ArrayIx2, mdRanges),
+                      SOA_COLUMN(ArrayIx2, segmentRanges),
+                      SOA_COLUMN(ArrayIx2, trackletRanges),
+                      SOA_COLUMN(ArrayIx2, tripletRanges),
+                      SOA_COLUMN(ArrayIx2, quintupletRanges),
+                      SOA_COLUMN(ArrayIx2, trackCandidateRanges))
```

Comment:
is `trackCandidateRanges` used? I recall that this collection is by itself, not binned in module bins

Reply:
Good point. I think it's not used.